Kitodo/src/main/java/org/kitodo/production/services/data/FilterField.java

```diff
@@ -0,0 +1,219 @@
+            "~.typeAutomatic = :queryObject AND ~.title",
+            "tasks AS task WITH task.typeAutomatic = :queryObject AND task.id", "typeAutomatic = :queryObject AND id",
+            Boolean.TRUE, "searchTask", "automatic"),
+    TASK_UNREADY("tasks AS task WITH task.processingStatus = :queryObject AND task.title",
+            "~.processingStatus = :queryObject AND ~.title",
+            "tasks AS task WITH task.processingStatus = :queryObject AND task.id",
+            "processingStatus = :queryObject AND id", TaskStatus.LOCKED, "searchTask", "locked"),
+    TASK_READY("tasks AS task WITH task.processingStatus = :queryObject AND task.title",
+            "~.processingStatus = :queryObject AND ~.title",
+            "tasks AS task WITH task.processingStatus = :queryObject AND task.id",
+            "processingStatus = :queryObject AND id", TaskStatus.OPEN, "searchTask", "open"),
+    TASK_ONGOING("tasks AS task WITH task.processingStatus = :queryObject AND task.title",
+            "~.processingStatus = :queryObject AND ~.title",
+            "tasks AS task WITH task.processingStatus = :queryObject AND task.id",
+            "processingStatus = :queryObject AND id", TaskStatus.INWORK, "searchTask", "inwork"),
+    TASK_FINISHED("tasks AS task WITH task.processingStatus = :queryObject AND task.title",
+            "~.processingStatus = :queryObject AND ~.title",
+            "tasks AS task WITH task.processingStatus = :queryObject AND task.id",
+            "processingStatus = :queryObject AND id", TaskStatus.DONE, "searchTask", "closed"),
+    TASK_FINISHED_USER(
+            "tasks AS task WITH task.processingStatus = :queryObject AND (task.processingUser.name = # OR task.processingUser.surname = # "
+                    .concat("OR task.processingUser.login = # OR task.processingUser.ldapLogin = #)"),
+            "~.processingStatus = :queryObject AND (~.processingUser.name = # OR ~.processingUser.surname = # "
+                    .concat("OR ~.processingUser.login = # OR ~.processingUser.ldapLogin = #)"),
+            "tasks AS task WITH task.processingStatus = :queryObject AND task.processingUser.id",
+            "processingStatus = :queryObject AND processingUser.id", TaskStatus.DONE, "searchTask", "closeduser");
+
+    /**
+     * Here the string search field names (user input) are mapped to the
+     * filters.
+     * 
+     * @param fieldName
+     *            user input string
+     * @return the constant
+     */
+    static FilterField ofString(String fieldName) {
+        if (Objects.isNull(fieldName)) {
+            return null;
+        }
+        switch (fieldName.toLowerCase()) {
+            case "": return null;
+            case "id": return PROCESS_ID;
+            case "parentprocessid":
+                return PARENT_PROCESS_ID;
+            case "process": return PROCESS_TITLE;
+            case "project": return PROJECT;
+            case "projectexact": return PROJECT;
```

Comment:
Should here not a return of `PROJECT_EXACT` instead of `PROJECT` as querying for an exact project title?

Reply:
@matthias-ronge : should this not fixed too?

Reply:
Yes. Fixed